enterprise/admin/__init__.py

```diff
@@ -18,8 +18,9 @@
 from enterprise.django_compatibility import reverse
 from enterprise.lms_api import CourseApiClient, EnrollmentApiClient
 from enterprise.models import (  # pylint:disable=no-name-in-module
-    EnterpriseCustomer, EnterpriseCustomerUser, EnterpriseCustomerBrandingConfiguration,
-    EnterpriseCustomerIdentityProvider, HistoricalUserDataSharingConsentAudit, PendingEnterpriseCustomerUser,
+    EnrollmentNotificationEmailTemplate, EnterpriseCustomer, EnterpriseCustomerUser,
+    EnterpriseCustomerBrandingConfiguration, EnterpriseCustomerIdentityProvider,
+    HistoricalUserDataSharingConsentAudit, PendingEnterpriseCustomerUser,
```

Comment:
Please feel free to log ENT project tickets for any tech debt you identify -- these can be used as filler-jobs for when there are small gaps at the ends of sprints.